src/pages/Home/Home.js

```diff
@@ -28,7 +28,7 @@ export default function Home(props) {
     const fetchMotorcycles = async () => {
       try {
         const res = await axios.get('/motorcycles.json');
-        const newMotorcycles = objectToArrayWithId(res.data).filter(motorcycle => motorcycle.status === 1);
+        const newMotorcycles = objectToArrayWithId(res.data).filter(motorcycle => motorcycle.status == 1);

```

Comment:
For statuses, you should create an object in a separated file like 
`MotorcycleStatus = { 
   Rented: 1; 
   Broken: 2; 
   InService: 3;
 }
`

and then in code, you should use "MotorcycleStatus.Rented" instead of "1". It's more descriptive.

Reply:
Probably 'motorcycle.status' is a string. i.e. '1' not 1. So in MotorcycleStatus you can use string as values like '1', '2', '3' and then restore '===' instead of '=='. Because on code reviews developers can correct you as well.

Reply:
You can use console.log(typeof motorcycle.status) to confirm whether this is a string.